test/unit/specs/components/Checkout/CheckoutBasketItem.spec.js

```diff
@@ -0,0 +1,116 @@
+import BasketItem from '@/components/Checkout/BasketItem';
+import kvAnalytics from '@/plugins/kv-analytics-plugin';
+import VueRouter from 'vue-router';
+import numeralFilter from '@/plugins/numeral-filter';
+import CookieStore from '@/util/cookieStore';
+import { render, within } from '@testing-library/vue';
+import loanReservation from '../../../fixtures/MatchedPromoLoanReservation.json';
+import basketLoanTeams from '../../../fixtures/BasketLoanTeams.json';
+
+// Test that the basket item renders the components it contains:
+// Checkout Item Img/Borrower Link, Remove Basket Item Button, Loan Reservation, Team Attribution, Loan Promo Credits,
+// Loan Price, Remove Basket Item, Matching Text
+
+const emptyComponent = {
+	template: '<div></div>',
+};
+
+describe('BasketItem loan', () => {
+	beforeEach(() => {
+	});
+	afterEach(jest.clearAllMocks);
```

Comment:
This `afterEach` shouldn't be needed in each of these files, as there aren't mocks to clear.